common/rpc/encryption/localStorePerHostCertProviderFactory.go

```diff
@@ -67,3 +69,26 @@ func (f *localStorePerHostCertProviderFactory) GetCertProvider(hostName string)
 
 	return cachedCertProvider, nil
 }
+
+func (f *localStorePerHostCertProviderFactory) Expiring(fromNow time.Duration,
+) (expiring CertExpirationMap, expired CertExpirationMap, errs []error) {
+
+	expiring = make(CertExpirationMap)
+	expired = make(CertExpirationMap)
+	errs = make([]error, 0)
+
+	if len(f.certProviderCache) == 0 {
+		return expiring, expired, errs
+	}
+
+	for _, provider := range f.certProviderCache {
+
+		providerExpiring, providerExpired, err := provider.Expiring(fromNow)
```

Comment:
rename to errs to indicates its plural

Reply:
also: using https://pkg.go.dev/github.com/hashicorp/go-multierror might be preferable to dealing with arrays of errors